Podfile

```diff
@@ -141,7 +141,7 @@ target 'WordPress' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :commit => 'aaa43ff63c6fc8dd44f6bd45af25e2f91619d6a5'
+    gutenberg :commit => '2bf7408427195ce361675a6bfb48d7ed07da22e3'
```

Comment:
Let's not forget this to update this as before merging
 gutenberg :tag => 'hotfix/v1.20.1'

Reply:
Thanks for the reminder. Updated as `:tag => 'v1.20.1'`.